problems/secondSmallest.js

```diff
@@ -6,8 +6,11 @@
 * @returns {number} - Returns the second smallest number.
 */
 
-function secondSmallest() {
-  
+function secondSmallest(arr = []) {
+    let secondSmallest = arr[0, 1, 2];
+    for(let i= 0; i < arr.length; i++) {
+        return secondSmallest
```

Comment:
If you return inside of a for loop the function will end. There for this will stop running during the first iteration.